app/jobs/create_institution_records_job.rb

```diff
@@ -26,11 +26,13 @@ def perform(institution_type_id)
           longitude: raw_institution.dig('geolokalizacja', 'longitude').to_f,
           website: raw_institution.fetch('stronaInternetowa'),
           email: raw_institution.fetch('email'),
-          city: raw_institution.fetch('adresDoKorespondecjiMiejscowosc'),
           street: raw_institution.fetch('adresDoKorespondecjiUlica'),
           building_no: raw_institution.fetch('adresDoKorespondecjiNumerBudynku'),
           apartment_no: raw_institution.fetch('adresDoKorespondecjiNumerLokalu'),
-          zip_code: raw_institution.fetch('adresDoKorespondecjiKodPocztowy')
+          zip_code: raw_institution.fetch('adresDoKorespondecjiKodPocztowy'),
+          powiat: raw_institution.fetch('powiat'),
+          gmina: raw_institution.fetch('gmina'),
+          miejscowosc: raw_institution.fetch('miejscowosc')
```

Comment:
So what's the final verdict when it comes to those names? I previously suggested two solutions: either using English names to stay consistent with the rest of the system or introducing an abstraction in the form of general administrative divisions.

Reply:
let's translate them to English